examples/c++/polynomial-api/example.cu

```diff
@@ -0,0 +1,286 @@
+#include <iostream>
+#define CURVE_ID BN254
```

Comment:
Should be defined at build time rather than hard coded i think

Reply:
Moved curve definition to cmake file